### DISCUSSION TITLE
Maybe wait until the new Xcode is actually released before complaining that it's out of date? :)

### DISCUSSION THREAD

**Discussion by mjreed-wbd** [Homebrew/discussions#58] [Everyday usage]
Homebrew now tells me that my version of Xcode is out of date, but when I try to update it I get the message that the software xcode-install is looking for is not available yet from Software Update. Whups!

 I'm assuming that both brew and xcode-install are using the same mechanism to determine what the "latest" version is, and that Apple may have updated what's being reported a little prematurely?

I don't know if there's a way to sanity-check what Apple is returning, but instructions which tell the user that if software-update doesn't find anything they should _delete their Xcode install and try from scratch_ seem unnecessarily dangerous when that might leave them without any Xcode at all or any easy way to reinstall it. (Sure, I can go get the GM 1 and download and install it manually, which will hopefully get me past the "outdated" warning.. but not everyone will know how to do that.. )

**Comment by SMillerDev** [comment]
This is actually a pretty common issue where Apple doesn't supply updates correctly. We only request people to update to stable versions, but sometimes those just don't show up in the apple software update system.
